rascal-lsp/src/main/rascal/library/util/LanguageServer.rsc

```diff
@@ -549,6 +650,9 @@ data DocumentSymbol
         set[DocumentSymbolTag] tags = {}
     );
 
+@description{
+    List of types that can be used for DocumentSymbol and CompletionItem.
```

Comment:
let's repeat the warning that some are specific to one and some are specific to the other.